src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Please, use singular form for enum names
```suggestion
public enum Color {

```